Gemfile

```diff
@@ -74,8 +75,8 @@ gem "omniauth-github"
 gem "cancancan"
 
 # Redis
-gem "redis"
-gem "redis-namespace"
+gem "redis", "~>4.1.0"
```

Comment:
Rails 6.1 will fixed this

https://github.com/rails/rails/commit/37c19f7ebcf542af968a4983b3296b9fa283a0dc#diff-60aba42516f795d80ed6f18ad14c1f6a